backend/lcfs/web/api/fuel_code/repo.py

```diff
@@ -217,13 +228,13 @@ async def get_fuel_type_by_id(self, fuel_type_id: int) -> FuelType:
     @repo_handler
     async def get_fuel_categories(self) -> List[FuelCategory]:
         """Get all fuel category options"""
-        return (await self.db.execute(select(FuelCategory))).scalars().all()
+        return (await self.db.execute(select(FuelCategory))).unique().scalars().all()
 
     @repo_handler
     async def get_fuel_category_by(self, **filters: Any) -> FuelCategory:
         """Get a fuel category by any filters"""
         result = await self.db.execute(select(FuelCategory).filter_by(**filters))
-        return result.scalar_one_or_none()
+        return result.unique().scalar_one_or_none()
```

Comment:
and again

Reply:
Thank you! I removed eager-loaded relationship.

Reply:
Looks like its still here?